### ISSUE TITLE
pytorch pretrained vgg

### ISSUE THREAD

**Issue by davidebaltieri31** [dingkeyan93/DISTS#5]
Not an issue per se, more like a suggestion.

I see you use pytorch pretrained vgg without correctly normalizing the image (all pytorch pretrained models assume the image was normalized with transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]) )

Retraining with the correct normalization might give you even better results.

**Comment by jzakirov** [comment]
They actually do normalize images in `forward_once` function, using `self.mean` and `self.std` constants
